llvm/docs/SecurityTransparencyReports.rst

```diff
@@ -2,7 +2,15 @@
 LLVM Security Group Transparency Reports
 ========================================
 
-This page lists the yearly LLVM Security group transparency reports.
+This page lists the yearly LLVM Security Response group transparency reports.
+
+The LLVM Security Response group started out as the LLVM security group, previous
+year's transparency reports keep the original year.
```

Comment:
maybe s/keep the original year/keep the original name/?

Reply:
Thanks for spotting. Done